upgrader/upgrade-062-070/scripts/upgradeKeptn.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+source ./utils.sh
+
+./upgradecollections $MONGODB_URL $CONFIGURATION_SERVICE_URL
+
+# Upgrade from Helm v2 to Helm v3
+helm init --client-only
+verify_install_step $? "Helm init failed."
+RELEASES=$(helm list -aq)
+verify_install_step $? "Helm list failed."
+echo $RELEASES
+
+helm3 plugin install https://github.com/helm/helm-2to3
+verify_install_step $? "Helm-2to3 plugin installation failed."
+yes y | helm3 2to3 move config
+verify_install_step $? "Helm-2to3 move of config failed."
+
+for release in $RELEASES; do
+  helm3 2to3 convert $release --dry-run
+  verify_install_step $? "Helm2-to3 release convertion dry-run failed"
+  helm3 2to3 convert $release
+  verify_install_step $? "Helm2-to3 release convertion failed"
+done
+
+yes y | helm3 2to3 cleanup --tiller-cleanup
+verify_install_step $? "Helm2-to3 cleanup failed"
+
+
+
+KEPTN_VERSION="release-0.7.0"
```

Comment:
As `release-0.7.0` is not available yet, the upgrader is hard to test.

Any change we can make `KEPTN_VERSION` an environment variable for the manifest that we apply? We could then use it as follows in the bash script:

```console
KEPTN_VERSION=${KEPTN_VERSION:-"release-0.7.0"}
```

and add an environment variable to the manifest `KEPTN_VERSION=master`